tests/functional/privacy/test_privacy_page_availability.py

```diff
@@ -0,0 +1,102 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+import sys
+
+import pytest
+import requests
+from redirects.base import assert_valid_url
+
+# With apologies, but tests/ is not a Python module and this avoids
+# duplicating the code for assert_valid_url
+sys.path.append("../../redirects")
```

Comment:
don't you have to do this before the import?

Reply:
It seems to work either way and linting pushes it down. However, will push it up and make `isort` ignore it

Reply:
does it work without this line then? If it works with this below the import then I'm guessing it should. Otherwise I'm very confused.

Reply:
Pulled it out and things run fine still. I've checked that I haven't got a stale, uncommitted `__init__.py` or any cached python around locally and it seems to work here, and in Circle, and in [integration tests](https://gitlab.com/mozmeao/www-config/-/pipelines/609407923). 

I can't recall exactly what file and import setup triggered it, but what we've got now def works now, with no hacks. 🪚